schedule/schedule_commands.go

```diff
@@ -656,7 +656,7 @@ func encodeMemo(memo map[string]interface{}) (*commonpb.Memo, error) {
 	if len(memo) == 0 {
 		return nil, nil
 	}
-	dc := dataconverter.CustomDataConverter()
+	dc := converter.GetDefaultDataConverter()
```

Comment:
Correctness: here we are in `encodeMemo()`. This helper function which is used only in `getMemoAndSearchAttributesForSchedule` which is used only in `CreateSchedule`: https://github.com/temporalio/cli/blob/80a17b4de02405e0edc646754b7d48094bf51095/schedule/schedule_commands.go#L230
 
There we are creating a request, which will be sent using `frontendClient`, which has the gRPC interceptor configured: https://github.com/temporalio/cli/blob/80a17b4de02405e0edc646754b7d48094bf51095/schedule/schedule_commands.go#L219